hw/ip/otbn/dv/otbnsim/sim/sim.py

```diff
@@ -27,10 +30,18 @@ def run(self, verbose: bool) -> int:
 
         '''
         insn_count = 0
+        # ISS will stall at start until URND data is valid, immediately set it
+        # valid when in free running mode as nothing else will.
+        self.state.set_urnd_reseed_complete()
         while self.state.running:
             self.step(verbose)
             insn_count += 1
 
+            if self.state.wsrs.RND.pending_request:
+                # Immediately make RND data available where an instruction
+                # requests it.
```

Comment:
Nit: This sounds a bit like the RND data is being put in the place where the instruction asked for it. Maybe "If an instruction requests RND data, make it available immediately"?